CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.8.9
+* Added support for iOS App Extensions:
+	* Now, if you add a macro to project settings (LTH_APP_EXTENSIONS), LTHPasscode doesn’t crash. Also, with a new method, you can provide the size of the view in which the LHTPasscode view is going to be presented.
```

Comment:
Can we mark `LTH_APP_EXTENSIONS` as code? And the same for `LTHPasscode`, but also renamed to `LTHPasscodeViewController`.

> you can provide _a view in which the lock is going to be presented and centered in_.